### PR TITLE
TypeScript-STL v1.0 & Samchon-Framework v2.0 Beta

### PR DESCRIPTION
**Pull request by samchon**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

TypeScript v1.0 is released.

Samchon Framework is upgraded from v1.2 to v2.0 beta

### CODE REVIEW

File: typescript-stl/typescript-stl.d.ts [hunk 1]
```diff
@@ -3629,8 +3622,8 @@ declare namespace std {
      * <p> All objects thrown by components of the standard library are derived from this class.
      * Therefore, all standard exceptions can be caught by catching this type by reference. </p>
      *
-     * <p> <a href="http://samchon.github.io/typescript-stl/api/assets/images/design/exceptions.png" target="_blank">
-     * <img src="http://samchon.github.io/typescript-stl/api/assets/images/design/exceptions.png" style="max-width: 100%" /> </a> </p>
+     * <p> <a href="D:/Homepage/samchon.github.io/typescript-stl/images/class_diagram/exceptions.png" target="_blank">
+     * <img src="D:/Homepage/samchon.github.io/typescript-stl/images/class_diagram/exceptions.png" style="max-width: 100%" /> </a> </p>
```
Comment (sandersn) [inline typescript-stl/typescript-stl.d.ts]: >D:/Homepage/ [](start = 17, length = 12)

this path looks incorrect now

Comment (sandersn) [inline typescript-stl/typescript-stl.d.ts]: and in other places in this file

---
In reply to: [74475584](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/10557#discussion_r74475584) [](ancestors = 74475584)

**Comment by samchon** [comment]
@sandersn Thanks for the review. I'll fix the comment erros and do pull-request again.

**Comment by samchon** [comment]
#10577 Revised.

**Pull request closed without merging**